src/ensembl/src/header/Header.scss

```diff
@@ -61,25 +61,14 @@
   }
 }
 
-.aboutEnsembl {
+.topRight {
   display: flex;
-  align-items: baseline;
-  margin-top: 4px;
-  font-size: 12px;
-  color: $light-blue;
-}
-
-.logotypeAbout {
-  height: 10px;
-  fill: $light-blue;
-  margin-left: 0.5rem;
-}
-
-// For disabling temporarily the about Ensembl link in production
-.aboutEnsemblDisabled {
-  color: $medium-dark-grey;
+  align-items: center;
+  margin-left: 20px;
```

Comment:
Why?

Reply:
No idea why  : ) i think i just copied from where it was before and didnt question it :) In fact even the display flex can go. Added the changes.